modules/darwin/src/main/ts/ephox/darwin/mouse/MouseSelection.ts

```diff
@@ -25,8 +25,15 @@ export const MouseSelection = (bridge: WindowBridge, container: SugarElement<Nod
       findCell(event.target, isRoot).each((finish) => {
         CellSelection.identify(start, finish, isRoot).each((cellSel) => {
           const boxes = cellSel.boxes.getOr([]);
-          // Wait until we have more than one, otherwise you can't do text selection inside a cell.
-          if (boxes.length > 1) {
+          if (boxes.length === 1) {
```

Comment:
Is this going to work if you have a `contenteditable="true"` element is inside a `contenteditable="false"` cell and then click into the editable cell? eg:
```
<td contenteditable="false"><p contenteditable="true">1</p></td>
```

Reply:
Thanks for picking this up. I have reworked this based on our discussions and added test case for it so hopefully this edge case is covered now.